llpc/test/shaderdb/gfx9/PipelineVsFs_test_cache_alpha_to_coverage_disabled.pipe

```diff
@@ -0,0 +1,276 @@
+colorBuffer[0].channelWriteMask = 15
+colorBuffer[0].blendEnable = 0
+colorBuffer[0].blendSrcAlphaToColor = 0
+colorBuffer[1].format = VK_FORMAT_R32G32B32A32_SFLOAT
+colorBuffer[1].channelWriteMask = 15
+colorBuffer[1].blendEnable = 0
+colorBuffer[1].blendSrcAlphaToColor = 0
+colorBuffer[2].format = VK_FORMAT_R32G32B32A32_SFLOAT
+colorBuffer[2].channelWriteMask = 15
+colorBuffer[2].blendEnable = 0
+colorBuffer[2].blendSrcAlphaToColor = 0
+nggState.enableNgg = 0
+nggState.enableGsUse = 0
+nggState.forceNonPassthrough = 0
+nggState.alwaysUsePrimShaderTable = 0
+nggState.compactMode = NggCompactSubgroup
+nggState.enableFastLaunch = 0
+nggState.enableVertexReuse = 0
+nggState.enableBackfaceCulling = 0
+nggState.enableFrustumCulling = 0
+nggState.enableBoxFilterCulling = 0
+nggState.enableSphereCulling = 0
+nggState.enableSmallPrimFilter = 0
+nggState.enableCullDistanceCulling = 0
+nggState.backfaceExponent = 0
+nggState.subgroupSizing = Auto
+nggState.primsPerSubgroup = 0
+nggState.vertsPerSubgroup = 0
+options.includeDisassembly = 0
+options.scalarBlockLayout = 0
+options.includeIr = 0
+options.robustBufferAccess = 0
+options.reconfigWorkgroupLayout = 0
+
+
+[VertexInputState]
+binding[0].binding = 0
+binding[0].stride = 16
+binding[0].inputRate = VK_VERTEX_INPUT_RATE_VERTEX
+binding[1].binding = 1
+binding[1].stride = 16
+binding[1].inputRate = VK_VERTEX_INPUT_RATE_VERTEX
+binding[2].binding = 2
+binding[2].stride = 16
+binding[2].inputRate = VK_VERTEX_INPUT_RATE_VERTEX
+binding[3].binding = 3
+binding[3].stride = 16
+binding[3].inputRate = VK_VERTEX_INPUT_RATE_VERTEX
+binding[4].binding = 4
+binding[4].stride = 16
+binding[4].inputRate = VK_VERTEX_INPUT_RATE_VERTEX
+attribute[0].location = 0
+attribute[0].binding = 0
+attribute[0].format = VK_FORMAT_R32G32B32A32_SFLOAT
+attribute[0].offset = 0
+attribute[1].location = 1
+attribute[1].binding = 1
+attribute[1].format = VK_FORMAT_R32G32B32A32_SFLOAT
+attribute[1].offset = 0
+attribute[2].location = 2
+attribute[2].binding = 2
+attribute[2].format = VK_FORMAT_R32G32B32A32_SFLOAT
+attribute[2].offset = 0
+attribute[3].location = 3
+attribute[3].binding = 3
+attribute[3].format = VK_FORMAT_R32G32B32A32_SFLOAT
+attribute[3].offset = 0
+attribute[4].location = 4
+attribute[4].binding = 4
+attribute[4].format = VK_FORMAT_R32G32B32A32_SFLOAT
+attribute[4].offset = 0
+;Compiler Options: /usr/local/google/home/stevenperron/llpc/amdvlk_proper/drivers/xgl/build.debug/llpc/amdllpc -auto-layout-desc -spvgen-dir=/usr/local/google/home/stevenperron/llpc/amdvlk_proper/drivers/xgl/build.debug/spvgen -gfxip=9.0.0 vert.spv frag.spv --enable-pipeline-dump -unroll-max-percent-threshold-boost=1000 -unroll-threshold=700 -unroll-partial-threshold=700 -pragma-unroll-threshold=1000 -unroll-allow-partial -simplifycfg-sink-common=false -amdgpu-vgpr-index-mode -amdgpu-atomic-optimizations -use-gpu-divergence-analysis -filetype=obj -shader-cache-file-dir=/usr/local/google/home/stevenperron/llpc/test/reloc/creator_test 
```

Comment:
please remove the compiler options and [CompileLog] section, it isn't necessary for test scripts